crystal-map-api/src/main/java/com/schwarz/crystalapi/schema/DefaultSchemaValidator.kt

```diff
@@ -16,9 +16,7 @@ open class DefaultSchemaValidator : SchemaValidator {
     }
 
     protected open fun validateModelLevel(current: EntitySchema?, released: EntitySchema, logger: SchemaValidationLogger): Boolean {
-
```

Comment:
One run of ktlint format, @sbra0902 are the empty lines after the function definitions desired? I could modify the ktlint rules if this is intended